### ISSUE TITLE
CachalotPanel no longer works. Error importing  _invalidate_tables in transaction.py

### ISSUE THREAD

**Issue by AlJohri** [BertrandBordage/django-cachalot#62]
adding `cachalot.panels.CachalotPanel` to `DEBUG_TOOLBAR_PANELS` causes: `django.core.exceptions.ImproperlyConfigured: Error importing debug panel cachalot.panels: "cannot import name '_invalidate_tables'"`.

Full traceback:

```
Traceback (most recent call last):
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/core/management/commands/runserver.py", line 113, in inner_run
    autoreload.raise_last_exception()
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/apps.py", line 12, in <module>
    from debug_toolbar.middleware import DebugToolbarMiddleware
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/middleware.py", line 17, in <module>
    from debug_toolbar.toolbar import DebugToolbar
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 151, in <module>
    urlpatterns = DebugToolbar.get_urls()
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 145, in get_urls
    for panel_class in cls.get_panel_classes():
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 122, in get_panel_classes
    (panel_module, e))
django.core.exceptions.ImproperlyConfigured: Error importing debug panel cachalot.panels: "cannot import name '_invalidate_tables'"
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x10d2cc158>
Traceback (most recent call last):
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 118, in get_panel_classes
    mod = import_module(panel_module)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/cachalot/panels.py", line 14, in <module>
    from .utils import _get_table_cache_key
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/cachalot/utils.py", line 18, in <module>
    from .transaction import AtomicCache
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/cachalot/transaction.py", line 40, in <module>
    from .utils import _invalidate_tables
ImportError: cannot import name '_invalidate_tables'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/core/management/commands/runserver.py", line 113, in inner_run
    autoreload.raise_last_exception()
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/apps.py", line 12, in <module>
    from debug_toolbar.middleware import DebugToolbarMiddleware
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/middleware.py", line 17, in <module>
    from debug_toolbar.toolbar import DebugToolbar
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 151, in <module>
    urlpatterns = DebugToolbar.get_urls()
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 145, in get_urls
    for panel_class in cls.get_panel_classes():
  File "/Users/myuser/.virtualenvs/myproj/lib/python3.5/site-packages/debug_toolbar/toolbar.py", line 122, in get_panel_classes
    (panel_module, e))
django.core.exceptions.ImproperlyConfigured: Error importing debug panel cachalot.panels: "cannot import name '_invalidate_tables'"
```

**Comment by BertrandBordage** [comment]
I can only reproduce this when `'cachalot'` is not in `INSTALLED_APPS`. Did you add it?

**Comment by BertrandBordage** [comment]
It only happens when django-cachalot is after django-debug-toolbar in `INSTALLED_APPS`.
This will be fixed in version 1.4.1.

**Comment by AlJohri** [comment]
ah, that makes sense. thanks @BertrandBordage!
